tensorflow/compiler/xla/service/hlo_tfgraph_builder_test.cc

```diff
@@ -178,6 +178,23 @@ TEST_F(HloTfGraphBuilderTest, EmbeddedComputationsDiamond) {
   EXPECT_GT(generator_.GetGraphDef().node_size(), 0);
 }
 
+TEST_F(HloTfGraphBuilderTest, TokenHasNoLayout) {
+  auto builder = HloComputation::Builder("Token");
+  auto token = builder.AddInstruction(HloInstruction::CreateToken());
+  OpMetadata metadata;
+  metadata.set_op_name("x");
+  metadata.set_op_type("y");
+  token->set_metadata(metadata);
+  TF_CHECK_OK(generator_.AddComputation(*builder.Build()));
+  GraphDef graph_def = generator_.GetGraphDef();
+  EXPECT_EQ(graph_def.node_size(), 1);
```

Comment:
This one is more of a nit, but since you have to change the above anyway: We'd prefer ASSERT_EQ so that if the graph has 0 nodes, this won't segfault the test.